tf2onnx/constants.py

```diff
@@ -27,7 +27,8 @@
 TARGET_RS5 = "rs5"
 TARGET_RS6 = "rs6"
 TARGET_CAFFE2 = "caffe2"
-POSSIBLE_TARGETS = [TARGET_RS4, TARGET_RS5, TARGET_RS6, TARGET_CAFFE2]
+TARGET_ONNXRUNTIME = "onnxruntime"
+POSSIBLE_TARGETS = [TARGET_RS4, TARGET_RS5, TARGET_RS6, TARGET_CAFFE2, TARGET_ONNXRUNTIME]
 DEFAULT_TARGET = []
```

Comment:
the logic in tf2onnx to add cast to node input is wrong, for not all input need to insert cast, for example slice input "start" doesn't need a cast.

while to avoid making this PR too large and make all test can pass in opset 10, i will quick fix rs6 and add a todo.